tasks-by-users/nehalskdev/02 Poject/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta charset="name" content="internet, google, youtube, blog, vlog">
+    <titl> <h1>Airborne Traveler ©</h1> </titl>
```

Comment:
`title`

>you don't have to use h1 in title

Reply:
gotcha. i have made changes as per your sugguestions please check and approve. thanks